test/babel-transform-undebug.mjs

```diff
@@ -0,0 +1,46 @@
+import path from 'path'
+import test from 'tape'
+import babel from '@babel/core'
+import transformConstants from '../script/babel-transform-undebug.mjs'
+
+function transformCode(code) {
+  const result = babel.transformSync(code, {
+    plugins: [transformConstants],
+    filename: path.resolve('test/babel-transform-undebug.mjs')
```

Comment:
```suggestion
    filename: path.resolve(path.join('test', 'babel-transform-undebug.mjs'))
```

windows

Reply:
AFAIK resolve normalizes slashes

Reply:
I thought it didn’t (I responded somewhere else earlier). I’ll add tests on Windows in Actions later, or you could if you want to check